src/expandableCalendar/agendaList.tsx

```diff
@@ -37,6 +50,8 @@ interface Props extends SectionListProps<any, DefaultSectionT> {
   avoidDateUpdates?: boolean;
   /** offset scroll to section */
   viewOffset?: number;
+  /** enable scrolling the agenda list to the next date with events while pressing a date without events */
```

Comment:
Will skip empty dates? Or scroll to them if they are the next date with content?

Reply:
will scroll to the next date with content.. I changed the description :)